releasetasks/templates/firefox/release_graph.yml.tmpl

```diff
@@ -150,11 +150,19 @@ tasks:
         {{ finalVerify_tasks()|indent(4) }}
     {% endif %}
 
-    {% if postrelease_bouncer_aliases_enabled or postrelease_version_bump_enabled %}
-        {% macro post_release_human_decision_tasks() %}
-            {% include "post_release_human_decision.yml.tmpl" %}
+
+    {% if postrelease_bouncer_aliases_enabled or postrelease_version_bump_enabled or publish_to_balrog_channels %}
+        {% macro publish_release_human_decision_tasks() %}
+            {% include "publish_release_human_decision.yml.tmpl" %}
+        {% endmacro %}
+        {{ publish_release_human_decision_tasks()|indent(4) }}
+    {% endif %}
+
+    {% if publish_to_balrog_channels %}
```

Comment:
I guess we could put this and the two conditions below within the `if postrelease_bouncer_aliases_enabled or postrelease_version_bump_enabled or publish_to_balrog_channels` condition block. just to be safe as we don't ever want to run those without human decision. for now at least